contrib/gen-pem-cert.sh

```diff
@@ -1,10 +0,0 @@
-#!/bin/sh
```

Comment:
Think you need to remove the call to this in fd_frank_init, and remove `certs.rs` as well.

Reply:
(And remove the args provided to `fd_frank_run` ?)

Reply:
Also update `default.toml`, the comment about the `scratch_directory` storing certificates is no longer correct